include/rpc/dispatcher.h

```diff
@@ -60,6 +60,19 @@ class dispatcher {
               detail::tags::nonvoid_result const &,
               detail::tags::nonzero_arg const &);
 
+    //! \brief removes a functor with given name from callable functors.
+	void remove(std::string const &name) {
```

Comment:
Any reason for naming this remove instead of unbind ?
Also, you could return a boolean telling the caller whether something was erased or not ?